include_core/AtomicSupport.hpp

```diff
@@ -227,7 +227,9 @@ class VM_AtomicSupport
 		asm volatile("lock orq $0x0,(%%rsp)" ::: "memory");
 #elif defined(ARM) /* defined(J9HAMMER) */
 		__sync_synchronize();
-#elif defined(S390) /* defined(ARM) */
+#elif defined(AARCH64) /* defined(ARM) */
+		__sync_synchronize();
```

Comment:
Is there any reason why we shouldn't merge ARM and AARCH64 into the same #ifdef condition for now since they end up doing the same thing?  i.e., `#elif defined(ARM) || defined (AARCH64)`?  @youngar @rwy0717

Reply:
I didn't have time to review everything here but aarch64 introduced new instructions. Down the road (once we can test) maybe there's a better way of doing it? I thought it'd be better to keep it separate since its also separate everywhere else. Looked more consistent

Reply:
sync_synchronize does full barrier but 
http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0802a/a64_general_alpha.html
shows more specific options. currently rewritting.

Reply:
OK.  I'm fine with leaving it the way you had it.  I was just asking the question in case the port lib experts wanted to chime in.